app/models/user.rb

```diff
@@ -1,8 +1,13 @@
 class User < ApplicationRecord
-  has_many :posts, class_name: 'Post', foreign_key: :author_id
-  has_many :comments, class_name: 'Comment', foreign_key: :author_id
-  has_many :likes, class_name: 'Like', foreign_key: :author_id
-  validates :name, presence: true
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly consider your application asking for confirmation by adding `:confirmable` as stated by the project requirements below.
>![image](https://github.com/imranhossainemi/blog-website/assets/53402600/4039901a-62c9-477a-aa45-7bb851ecb77e)

Reply:
@omarsalem7  how the f I am going confirm email and without sent or receiving email for confirmation